firefox-ios/Client/Frontend/NativeErrorPage/NativeErrorPageViewController.swift

```diff
@@ -89,37 +89,67 @@ final class NativeErrorPageViewController: UIViewController,
 
     private var contraintsList = [NSLayoutConstraint]()
 
-    required init?(
-        coder aDecoder: NSCoder
-    ) {
-        fatalError(
-            "init(coder:) has not been implemented"
-        )
-    }
-
     init(
-        model: ErrorPageModel,
         windowUUID: WindowUUID,
         themeManager: ThemeManager = AppContainer.shared.resolve(),
         overlayManager: OverlayModeManager,
         notificationCenter: NotificationProtocol = NotificationCenter.default
     ) {
-        self.model = model
         self.windowUUID = windowUUID
         self.themeManager = themeManager
         self.overlayManager = overlayManager
         self.notificationCenter = notificationCenter
+        nativeErrorPageState = NativeErrorPageState(windowUUID: windowUUID)
+
         super.init(
             nibName: nil,
             bundle: nil
         )
 
+        subscribeToRedux()
         configureUI()
         setupLayout()
         adjustConstraints()
         showViewForCurrentOrientation()
     }
 
+    // MARK: Redux
+    func newState(state: NativeErrorPageState) {
+//        if state.title != nil {
```

Comment:
?? what dis?

Reply:
Thank you for pointing this out. This will be removed.